core/src/cpus/cortex_m/shared_types.zig

```diff
@@ -0,0 +1,310 @@
+//!
+//! This file contains types shared between different CPUs
+//!
+
+const builtin = @import("builtin");
+
+pub const CpuFlags = struct {
+    has_hard_fault: bool = true,
+    has_bus_fault: bool,
+    has_usage_fault: bool,
+    has_mem_manage_fault: bool,
+};
+
+pub const options = struct {
+    pub const Basic_Options = struct {
+        /// If true, the Cortex-M interrupts will be initialized with a more verbose variant
+        /// of the interrupt handlers which print the interrupt name.
+        ///
+        /// NOTE: This option is enabled in debug builds by default.
+        verbose_unhandled_irq: bool = (builtin.mode == .Debug),
+    };
+
+    pub const Ram_Vector_Options = struct {
+        /// When true, interrupt vectors are moved to RAM so handlers can be set at runtime.
+        ram_vectors: bool = false,
+        /// When true, the vector table lives in RAM.
+        ram_vector_table: bool = false,
```

Comment:
I moved your field here @tact1m4n3 , not sure how it works with the above one (`ram_vectors`)